DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: vitalsigns
+Title: A Utility Package for Conservation International's Vital Signs Project 
+Version: 0.1
+Authors@R: person("Thomas", "Johnson", email = "thomascjohnson@gmail.com",
```

Comment:
Added the SystemRequirements, check the new commit to check the language, or see it here: 

    The user will need to have AWS CLI installed. This can be found at https://aws.amazon.com/cli/ for Windows, Mac and Linux.